skype/skype-ps/skype/New-CsTenantNetworkSite.md

```diff
@@ -20,7 +20,7 @@ As an Admin, you can use the Windows PowerShell command, New-CsTenantNetworkSite
 ### Identity (Default)
 ```
 New-CsTenantNetworkSite [-Tenant <System.Guid>] [-Description <String>] [-NetworkRegionID <String>]
- [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-OnlineVoiceRoutingPolicy <String>]
+ [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-EmergencyCallRoutingPolicy <Object>] [-EmergencyCallingPolicy <Object>] [-OnlineVoiceRoutingPolicy <String>]
```

Comment:
```suggestion
 [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-EmergencyCallRoutingPolicy <String>] [-EmergencyCallingPolicy <String>] [-OnlineVoiceRoutingPolicy <String>]
```

Why not string? I think we always (or almost) use String for the policy name.

Reply:
PlatyPS put Object but is true that is better String here.